module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -512,6 +555,14 @@ public function isLoggedIn()
                 $results = $this->userTable->createRow();
                 $results->exchangeArray($this->session->userDetails);
                 $this->currentUser = $results;
+            } elseif ($this->cookieManager->get('loginToken')) {
```

Comment:
Do we ever want to attempt this if persistent login is disabled or inapplicable? Would it make sense to do a check first? For example, what if tokens exist out in the wild, but the administrator decides to turn off the feature. Shouldn't that stop the tokens from being used?

Reply:
Surely tokens should stop working when the feature is turned off. I added a check for this, but a couple rows below to make sure it works properly with ChoiceAuth. So if possible, we check the users auth method and then check whether persistent login is supported.